keps/1282-pods-ready-requeue-strategy/README.md

```diff
@@ -208,6 +208,22 @@ type RequeueState struct {
 }
 ```
 
+Add a new field `waitForPodsReadyStats` to the WorkloadStatus struct to track the number of evictions by reason:
+
+```go
+type WorkloadStatus struct {
+    // waitForPodsReadyStats holds statistics for WaitForPodsReady requeues.
+    // +optional
+    WaitForPodsReadyStats *WaitForPodsReadyStats `json:"waitForPodsReadyStats,omitempty"`
+}
+
+type WaitForPodsReadyStats struct {
+    // evictionByReasonCount maps eviction reasons to their counts.
```

Comment:
What other fields do you expect to have here in the future?

Reply:
no specific expectation at the moment, it was proposed as a potential container since WaitForPodsReady is a growing feature we may anticipate more metrics to track.

However, I'm totally ok to inline the field as `WaitForPodsReadyEvictionByReasonCount`. We can always revisit in v1beta2.